tests/util/bft.py

```diff
@@ -165,14 +175,28 @@ def stop_all_replicas(self):
 
         self.procs = {}
 
+    def start_replicas(self, replicas):
+        """
+        Start from list "replicas"
+        """
+        for r in replicas:
```

Comment:
If you want to be more "pythonistic", you could say:
[self.start_replica(r) for r in replicas]